.github/workflows/continuous-integration.yml

```
@@ -9,5 +9,6 @@ jobs:
   ci:
     uses: ray-di/.github/.github/workflows/continuous-integration.yml@v1
     with:
-      old_stable: '["7.2", "7.3", "7.4", "8.0", "8.1", "8.2"]'
-      current_stable: 8.3
+      old_stable: '["7.2", "7.3", "7.4", "8.0", "8.1", "8.2", "8.3"]'
+      current_stable: 8.4
+c
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 14: could not find expected ':'

(syntax-check)